package.json

```diff
@@ -46,6 +41,7 @@
   "devDependencies": {
     "@balena/contrato": "^0.6.0",
     "@balena/es-version": "^1.0.1",
+    "@balena/happy-eyeballs": "0.0.6",
```

Comment:
Shouldn't this remain in dependencies? See: https://balena.fibery.io/Work/Pattern/Implement-Happy-Eyeballs-Across-All-Balena-Services-(device-status-is-VPN-Only-)-2877

Reply:
It is, I just moved it to "devDependencies"